docs/proposals/20201020-capi-provider-operator.md

```diff
@@ -754,36 +760,40 @@ will look like.
 
 ##### Operator Installation
 
-- `clusterctl init` will install the operator and its corresponding CRDs as a
-  pre-requisite if the operator doesn’t already exist. Please note that this
-  command will consider image overrides defined in the local clusterctl config
-  file.
-- If the admin does not want to use clusterctl to install the operator, it is
+- During the first phase of implementation `clusterctl` operations will not be 
+  supported and admin will have to install the operator manually, or in case if
+  the admin does not want to use clusterctl to install the operator, it is
   possible to `kubectl apply` the operator yaml that will be published in the
   cluster-api release artifacts.
+- In future `clusterctl init` will install the operator and its corresponding CRDs 
+  as a pre-requisite if the operator doesn’t already exist. Please note that this
+  command will consider image overrides defined in the local clusterctl config
+  file.
 
 ##### Operator Upgrade
-
-- The admin can use `clusterctl upgrade operator` to upgrade the operator
-  components. Please note that this command will consider image overrides
-  defined in the local clusterctl config file. Other commands such as
-  `clusterctl upgrade apply` will also allow to upgrade the operator.
-- `clusterctl upgrade plan` will identify when the operator can be upgraded by
-  checking the cluster-api release artifacts.
-- If the admin doesn’t want to use clusterctl, they can use kubectl apply with
+- During the first phase of implementation `clusterctl` operations will not be 
+  supported and admin will have to install the operator manually, or in case if
+  the admin doesn’t want to use clusterctl, they can use kubectl apply with
```

Comment:
Suggesting a rephrase similar to above (in case if the admin doesn’t want to use clusterctl does not make sense to me at this stage, also we are not going to publish operator artefacts in main CAPI release artefacts given that this requires release coordination and this can slow down both sides), with the difference that we are talking about upgrade.

Reply:
Also, doe you have any consideration about transitioning from manually managed operator to clusterctl managed operation? It will be great if we make it clear if this is something we want to support, and possible make it easier and smooth for the users, otherwise this will require additional upgrade instructions we should plan for (or at lease document as a TODO)

Reply:
You are right about upgrade instructions, I will document them as TODO for now.